apis/src/websockets/ws_connection.rs

```diff
@@ -135,8 +136,10 @@ impl StreamHandler<Result<ws::Message, ws::ProtocolError>> for WsConnection {
                 ctx.stop();
             }
             Ok(ws::Message::Nop) => (),
-            Ok(Text(s)) => {
-                if let Ok(request) = serde_json::from_str::<ClientRequest>(s.as_ref()) {
+            Ok(ws::Message::Binary(s)) => {
+                let request: Result<String,_> = MsgpackSerdeCodec::decode(&s);
+                let request = serde_json::from_str::<ClientRequest>(request.unwrap().as_ref());
```

Comment:
1. We can't hard unwrap here, we need to do an `if let Some` or something like that.
2. Why are we double encoding the messages now? First as JSON and then on top of that as MsgPack. That doesn't make sense to me, we should only encode them once. 

Why I am so strict here is because people will still have the old wasm running on their devices and they will still send with the old code and it will fail the unwrap here.